single_kernel_mongo/managers/mongodb_operator.py

```diff
@@ -684,7 +709,11 @@ def assert_pass_password_checks(self) -> None:
             raise NonDeferrableFailedHookChecksError(
                 "Cannot set password on shard, please set password on config-server."
             )
-        if isinstance(self.backup_manager.get_status(), MaintenanceStatus):
+
+        # todo future work - check status of pbm directly
+        pbm_statuses = self.backup_manager.get_statuses()
+        pbm_status = next(iter(pbm_statuses), None)
+        if isinstance(pbm_status, MaintenanceStatus):
```

Comment:
don't we already have `_get_backup_restore_operation_result` to get the information from?